maintainers/maintainer-list.nix

```diff
@@ -9753,6 +9753,12 @@
     githubId = 116740;
     name = "Paweł Pacana";
   };
+  paulg = {
```

Comment:
Could you put this in a seperate commit? Formatting for the description should be something like `maintainers: add paulg`. Then you can add the maintainer addition in `shipwright/default.nix` in the same commit adding the patches.